documentation/src/docs/deployment/run-local.md

```diff
@@ -76,9 +79,13 @@ HOST_IP=1.2.3.4 docker-compose up
 ```
 Clients can then connect to the controller at `${HOST_IP}:9090`.
 
-## Configuring standalone
+## Running Pravega in Standalone Mode with SSL/TLS Enabled
 
-Configure standalone server to communicate using SSL/TLS. To do so, edit the TLS-related properties in `standalone-config.properties` as shown below:
+By default the `singlenode.enableTls` (SSl/TLS) is disabled. To enable it, set the value of  `singlenode.enableTls` to **True**. Once the `enableTls` (TLS) is enabled, the default certificates provided in the `conf` directory are used for setting up TLS.
```

Comment:
Can we rephrase this to something like: SSL/TLS is disabled by default in Pravega standalone mode cluster.

Also, item 1 in the points below already explains how to enable it, so the following statements are redundant: "To enable it, set the value of  `singlenode.enableTls` to **True**. Once the `enableTls` (TLS) is enabled, the default certificates provided in the `conf` directory are used for setting up TLS." We could say something like the following instead:

"The steps below show how to enable and run standalone mode cluster with SSL/TLS enabled:"

Reply:
@ravisharda Accepted suggestion